osbs/cli/main.py

```diff
@@ -135,7 +135,8 @@ def cmd_list_builds(args, osbs):
                             key=lambda x: x.get_time_created_in_seconds()):
             unique_image = build.get_image_tag()
             try:
-                image = strip_registry_from_image(build.get_repositories()["primary"][0])
+                image = \
+                    ImageName.parse(build.get_repositories()["primary"][0]).to_str(registry=False)
             except (TypeError, KeyError, IndexError):
```

Comment:
Are these exception still valid when `ImageName` is used?

Reply:
I guess since we are passing (build.get_repositories()["primary"][0])
seems to me like all could happen